### ISSUE TITLE
LSIO docker container plugin install

### ISSUE THREAD

**Issue by kevindd992002** [springjools/deluge-autoremoveplus#21]
I'm using LSIO deluge container and I have the python3-requests module. But when I try to install using the webUI, I see the name of the egg as [object FileList] when I choose it. It's not working this way but when I manually install by copying to the plugins folder, it works just fine. So why can't we install using the UI?

**Comment by springjools** [comment]
No idea, I've always installed the manual way. I'll investigate.

**Comment by springjools** [comment]
The same thing happens for me. Not sure if it just affects this plugin or all deluge plugins. The issue is mentioned before with both this plugin and the webui plugin. 

Instead of copying the .egg to the right folder, deluge web-install somehow creates a file called object list instead.

**Comment by springjools** [comment]
See https://github.com/idlesign/deluge-webapi/issues/14

**Comment by springjools** [comment]
Maybe this fixes it: https://github.com/deluge-torrent/deluge/pull/284/commits/15c250e15252fb9131a1b4c39226fcd43d45fc08

**Comment by kevindd992002** [comment]
Oh ok. So this means that the issue in Deluge itself and not the autoremoveplus plugin?

**Comment by mikkihugo** [comment]
How did you get 0.6.15 to work in LSIO docker? I tried renaming it to py3.6 but it doesent even appear in plugin list.

**Comment by kevindd992002** [comment]
Did you use the GUI to install it? If so, then it won't work and is the whole point of this thread. You have to manually copy it in the plugins folder like so:

![image](https://user-images.githubusercontent.com/3907957/81116948-dc0db000-8f58-11ea-8ea5-926b102e4956.png)

**Comment by kevindd992002** [comment]
> 
> 
> Oh ok. So this means that the issue in Deluge itself and not the autoremoveplus plugin?

@springjools Any comment on this?

**Comment by mikkihugo** [comment]
> Did you use the GUI to install it? If so, then it won't work and is the whole point of this thread. You have to manually copy it in the plugins folder like so:
> 
> ![image](https://user-images.githubusercontent.com/3907957/81116948-dc0db000-8f58-11ea-8ea5-926b102e4956.png)

Thanks for the screenshot showing you were using the older  .14 - I downloaded that version and it worked :)

**Comment by kevindd992002** [comment]
Yeah, I just saw that there is a new version now. Are you saying that .15 is not working for you using the same manual install process?

**Comment by mikkihugo** [comment]
I downloaded the .15 egg and it did not work for me on latest LSIO deluge docker. I tried renaming to .36. and compiled myself with no luck.

**Comment by alekc** [comment]
Yep, can confirm the issue with docker and 0.6.15, even with manual installation it doesn't appear.

**Comment by kevindd992002** [comment]
@springjools Any ideas why the latest version of the plugin doesn't work? I'm still using .14 but I want to be able to upgrade to the latest version without breaking anything.
